packages/main/src/SegmentedButton.ts

```diff
@@ -136,17 +141,17 @@ class SegmentedButton extends UI5Element {
 	}
 
 	onEnterDOM() {
-		ResizeHandler.register(this.parentNode, this._handleResizeBound);
+		ResizeHandler.register(this.parentNode as HTMLElement, this._handleResizeBound);
```

Comment:
The _handleResizeBound method is async with signature "() => Promise<void>" while Resizehandler.register expects a callback with signature "() => void".

We will discuss if we should change the core ResizeHandler class in the core team.

Reply:
async handlers are now allowed - the change in the ResizeHandler has been merged